installer/datafiles/linux.data

```diff
@@ -199,12 +199,15 @@ fi
 # Ensure omsagent is in the omiusers group, but leave omsagent as a group
 /usr/sbin/usermod -g omiusers omsagent 1> /dev/null 2> /dev/null
 
-%Postinstall_200
+%Postinstall_500
 #include Syslog_Services
 #include Sudoer_Functions
 
 ${{OMI_SERVICE}} reload
-${{OMS_SERVICE}} enable
+if [ -f /etc/opt/microsoft/omsagent/conf/omsadmin.conf ]; then
+    ${{OMS_SERVICE}} enable
+fi
```

Comment:
If you're trying to get OMS to not start, this probably isn't it. Would you rather just abort the installation?